src/utilities/testHelpers.ts

```diff
@@ -14,7 +14,7 @@ export const component = { template: '<div>This is component</div>' }
 export const routes = createRoutes([
   {
     name: 'parentA',
-    path: '/[paramA]',
+    path: '/parentA/[paramA]',
```

Comment:
I update this route (and associated tests) because the path for `parentB` is `/parentB` which matched `/[paramA]`. Discovered while writing tests for `createRouterFind`.